requests-for-comments/2022-06-10-nail-exports.md

```diff
@@ -0,0 +1,110 @@
+# Request for comments: Nail Exports - Ben White
+
+## Problem statement
+
+More context here https://github.com/PostHog/product-internal/pull/308/
+
+
+### What can be Exported
+
+* Insights
+  - Visualisation - web, image
+  - Data - API, csv
+* Dashboards
+  - Visualisation - web, image (or collection of images)
+  - Data - API, csv
+* Persons, Cohorts, Groups
+
+### How can they be Exported
+* Embeds 
+  - Via an iframe or directly linked to a dedicated webpage for viewing the element 
+* Images
+  - Server-side rendering of the shared webpage (using chromedriver) to an image (.png)
+  - Possibly needs multiple dimension options
+* Subscriptions - email:
+  - User configured regular reports via email including exported image of the relevant item
+* Subscriptions - slack:
+  - Slack notification including the exported image(s)
+* CSV / Spreadsheet:
+  - Relevant data exported to a CSV / Spreadsheet
+* Custom Components (React)
+  - Integrated directly into customer's code, calling Posthog API for the data 
+* Alerts (variation on subscriptions):
```

Comment:
Note this feature seems unrelated to the rest of this RFC and not sure how this is achieved. Should this be tracked/solved separately?

Reply:
Only mentioned it here as one of the things we are trying to achieve with this RFC is to help pick a technical solutions that will support future use-cases and not back is into multiple unique solutions per-case 👍  
```suggestion
* Alerts (variation on subscriptions): _(out of scope for current exporting work)_
```